lib/internal/child_process.js

```diff
@@ -29,7 +29,11 @@ const {
     ERR_MISSING_ARGS
   }
 } = require('internal/errors');
-const { validateString, validateOneOf } = require('internal/validators');
+const {
+  validateArray,
+  validateString,
+  validateOneOf
```

Comment:
nit: ASCII order would be nice (and trailing comma ftw)
```suggestion
  validateOneOf,
  validateString,
```